Healthy/Classes/Modules/SavedRecipes/ MyTableViewCell.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+
+class MyTableViewCell: UITableViewCell {
+    // swiftlint:disable all
+    private let myImageView = UIImageView()
+    private let titleLabel = UILabel()
+    private let subtitle1Label = UILabel()
+    private let subtitle2Label = UILabel()
```

Comment:
We can then rename this to `chefNameLabel` for better readability and coherence.